.travis/script.sh

```diff
@@ -7,8 +7,9 @@ export HTTPIE_CONFIG_DIR=$TRAVIS_BUILD_DIR/.travis
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
 export PULP_CONTENT_ORIGIN=http://localhost:24816
 export PULP3_HOST=localhost:24817
-django-admin makemigrations file --noinput
+django-admin makemigrations --noinput
```

Comment:
Should we really need this command? I think we are shipping the migrations for some time now.

Reply:
I didn't mean to push it, I was doing other tests